internal/namespace/manager.go

```diff
@@ -937,3 +974,31 @@ func (nm *namespaceManager) getEventPlugins(ctx context.Context) (plugins map[st
 	}
 	return plugins, err
 }
+
+func (nm *namespaceManager) getAuthPlugin(ctx context.Context) (plugins map[string]authPlugin, err error) {
+	plugins = make(map[string]authPlugin)
+
+	authConfigArraySize := authConfig.ArraySize()
+	for i := 0; i < authConfigArraySize; i++ {
+		config := authConfig.ArrayEntry(i)
+		name, pluginType, err := nm.validatePluginConfig(ctx, config, "auth")
+		if err != nil {
+			return nil, err
+		}
+
+		plugin, err := authfactory.GetPlugin(ctx, pluginType)
+		if err != nil {
+			return nil, err
+		}
+
+		plugins[name] = authPlugin{
+			config: config.SubSection(pluginType),
+			plugin: plugin,
+		}
+	}
+	return plugins, err
+}
+
+func (nm *namespaceManager) Authorize(ctx context.Context, authReq *fftypes.AuthReq) error {
```

Comment:
Who calls this? I see that the routes in the apiserver also call directly to orchestrator.

Reply:
Oh looks like websocketConnection calls here?